File: app/src/main/java/com/example/epolisplusapp/models/error_models/InputPhoneNumberFailure.java
package com.example.epolisplusapp.models.error_models;


import android.content.Context;

import com.example.epolisplusapp.R;

class InputPhoneNumberFailure extends Failure {
    @Override
    String getErrorMessage(Context context) {
        return context.getString(R.string.input_phone_number_failure);
    }
}

File: app/src/main/java/com/example/epolisplusapp/models/error_models/InputPhoneNumberFailure.kt
package com.example.epolisplusapp.models.error_models

import android.content.Context
import com.example.epolisplusapp.R

class InputPhoneNumberFailure: Failure() {
    override fun getErrorMessage(context: Context): String {
        return context.getString(R.string.phone_fail)
    }
}